portal-wire-protocol.md

```diff
@@ -0,0 +1,158 @@
+# Portal Wire Protocol
+
+The Portal wire protocol is the base p2p protocol by which nodes communicate.
+
+The different Portal networks **MAY** use this base protocol, but they **MUST** remain separated per network.
+This is done at the [Node Discovery Protocol v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md#talkreq-request-0x05) layer, by providing a different protocol bytestring, per network, in the `TALKREQ` message.
+
+The protocol bytestring in the `TALKREQ` message **MUST** have "portal-" as prefix then followed by the network name (e.g. "portal-state").
+
+Each network can specify which specific messages from the protocol it supports.
```

Comment:
I think we should go a bit further here to define what implementations should do.

1. Each sub-protocol **MUST** specificy which messages are supported.
2. Unsupported messages should receive a TALKRESP message with an empty payload (or should we put something in there to allow differentiation from a client that just doesn't support the protocol like an error string..)